customer-config/checkout-config/che.tender.v1-preview.json

```diff
@@ -112,7 +112,17 @@
     "RestrictionRule": {
       "description": "A rule that is set on a Tender to specify which items can be purchased with this tender.",
       "type": "object",
+      "required": [
+        "id"
```

Comment:
Do we need them to be required?

Reply:
Yeah, we generally should enforce ids so it is easier for us in a future to modify objects with patch